includes/genericBackupTaker.class.php

```diff
@@ -161,18 +161,18 @@ function takeFullBackupSnapshot(backupJob $job, $snapshotGroup) {
 			
 					// Build the command...
 					$xbCommand = 'ssh -o StrictHostKeyChecking=no -p '.$hostInfo['ssh_port'].' '.$sbInfo['backup_user'].'@'.$hostInfo['hostname'].
-								" 'cd $tempDir ; innobackupex --ibbackup=".$xbBinary." --stream=xbstream ".$sbInfo['datadir_path']." --user=".$sbInfo['mysql_user'].
+								" 'cd $tempDir ; xtrabackup --backup --ibbackup=".$xbBinary." --stream=xbstream --target-dir=".$sbInfo['datadir_path']." --user=".$sbInfo['mysql_user'].
```

Comment:
> I cannot find ` --ibbackup=` option in xtrabackup documentation.

For xtrabackup 2.4 this option is still present since the binary call still be called through  the `innobackupex` name. For xtrabacku 8.0 this option is simply ignored.
 
>     * https://www.percona.com/doc/percona-xtrabackup/8.0/howtos/recipes_xbk_full.html tells to run the `--prepare` step twice. We can discuss about it if needed.

According to the https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/full_backup.html#creating-a-backup only one `--prepare` is necessary,

Reply:
> For xtrabackup 2.4 this option is still present since the binary call still be called through the innobackupex name. For xtrabacku 8.0 this option is simply ignored.

But you have replaced the `innobackupex` call by `xtrabackup` for all xbm versions, so why keep it then?

> only one --prepare is necessary

Fare enough

Reply:
> > For xtrabackup 2.4 this option is still present since the binary call still be called through the innobackupex name. For xtrabacku 8.0 this option is simply ignored.
> 
> But you have replaced the `innobackupex` call by `xtrabackup` for all xbm versions, so why keep it then?

No particular reason, i'll remove it then.

> > only one --prepare is necessary
> 
> Fare enough

My understanding of the documentation is that the second `--prepare` is used to ensure all binlog have been replayed